cmd/soroban-rpc/internal/config/toml.go

```diff
@@ -0,0 +1,124 @@
+package config
+
+import (
+	"fmt"
+	"io"
+	"reflect"
+	"strings"
+	"unicode"
+	"unicode/utf8"
+
+	"github.com/pelletier/go-toml"
+)
+
+func parseToml(r io.Reader, strict bool, cfg *Config) error {
+	tree, err := toml.LoadReader(r)
+	if err != nil {
+		return err
+	}
+
+	validKeys := map[string]struct{}{}
+	for _, option := range cfg.options() {
+		key := option.getTomlKey()
```

Comment:
I think that the "-" is more error prone then "" ( which is the default value ).
Just allowing the option to omit the toml key is shorter and less error prone.

Reply:
Using `""` means that the default would be for every config option to be excluded from the toml file. I wanted the default to be that everything is included, unless explicitly omitted..

Reply:
in that case, I have another request  : 
write a unit test that using reflection to iterate on all the public fields of Config and make sure all of them have a valid and unique toml key.